digital_land/commands.py

```diff
@@ -887,3 +898,40 @@ def organisation_check(**kwargs):
     lpa_path = kwargs.pop("lpa_path")
     package = OrganisationPackage(**kwargs)
     package.check(lpa_path, output_path)
+
+
+def download_categorical_fields(dataset):
+    base_url = "https://files.planning.data.gov.uk/dataset/"
+    specification = Specification("specification/")
+
+    try:
+        category_fields_query = specification.get_category_fields_query()
+    except KeyError as e:
+        print(f"Error in getting category fields query: {e}")
+        return
+
+    combined_matches = category_fields_query[
+        (category_fields_query["dataset"] == dataset)
+        | (category_fields_query["field"] == dataset)
+        | (category_fields_query["field-dataset"] == dataset)
+    ].drop_duplicates(subset=["dataset", "field", "field-dataset"])
+
+    if combined_matches.empty:
+        return
+
+    for _, row in combined_matches.iterrows():
```

Comment:
Does this still need to be in it looks like you've got this logic in the API call anyway